### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by OleksandrFEdev**

### CODE REVIEW

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Nice work, although some code refactoring is needed

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form "
+    data-qa="big"
+    action="#"
+    method="post">
```
Comment (Viktor-Kost) [inline src/index.html:18]: 1) apply code styling recommendations. For similar cases as well
2) remove extra space class="form " -> class="form"
```suggestion
    <form
      class="form"
      data-qa="big"
      action="#"
      method="post"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,36 @@
...
+      <label class="label" for="input">
+        <input
+        type="text"
+        name="input"
+        class="input"
+        id="input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      </label>
```
Comment (Viktor-Kost) [inline src/index.html:28]: no needs to use for/id if you wrap input with label

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+form {
```
Comment (Viktor-Kost) [inline src/style.css:30]: it's better to style classes instead of tags. Apply for similar cases

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (YegorKorenkov) [inline src/style.css:30]: Not fixed, please move all rules to .form selector
image

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  background-repeat: no-repeat;
+  outline: none;
+}
+
+.input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+input {
```
Comment (YegorKorenkov) [inline src/style.css:64]: Not fixed
image

File: src/index.html [hunk 5]
```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form"
```
Comment (YegorKorenkov) [inline src/index.html:15]: Please check formatting for each case
image

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 6]
```diff
@@ -8,9 +8,40 @@
...
+      data-qa="big"
+    >
+      <label>
+        <input
+        type="text"
+        class="search-city search-city-big"
+        name="searchCity"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
```
Comment (IvanFesenko) [inline src/index.html:27]: Pay attention on indents
```suggestion
        <input
          type="text"
          class="search-city search-city-big"
          name="searchCity"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```

File: src/index.html [hunk 7]
```diff
@@ -8,9 +8,40 @@
...
+    class="search-form"
+    data-qa="small"
+  >
+      <label>
+        <input
+        type="text"
+        class="search-city search-city-sm"
+        name="searchCity"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
```
Comment (IvanFesenko) [inline src/index.html:43]: Same here
```suggestion
        <input
          type="text"
          class="search-city search-city-sm"
          name="searchCity"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**